<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>571</width>
    <height>606</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);
background-color: rgb(255, 170, 127);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="enviacao">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>510</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(184, 184, 184);</string>
    </property>
    <property name="text">
     <string>Cadastrar</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>200</y>
      <width>50</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
background-color: rgb(255, 170, 127);
border-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>NOME: </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="nome">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>200</y>
      <width>300</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>280</y>
      <width>61</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
background-color: rgb(255, 170, 127);
border-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>EMAIL:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="email">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>280</y>
      <width>300</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>360</y>
      <width>91</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
background-color: rgb(255, 170, 127);
border-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>TELEFONE:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="telefone">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>360</y>
      <width>300</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QRadioButton" name="residencial">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>430</y>
      <width>95</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Residencial</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="celular">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>430</y>
      <width>95</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Celular</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnConsultar">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>510</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(184, 184, 184);</string>
    </property>
    <property name="text">
     <string>Consultar</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>20</y>
      <width>191</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 26pt &quot;Magneto&quot;;</string>
    </property>
    <property name="text">
     <string>Agenda</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>120</y>
      <width>50</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
background-color: rgb(255, 170, 127);
border-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>ID:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="id">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>120</y>
      <width>300</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>571</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
